doc/locale/ja/LC_MESSAGES/install/cmake_options.po

```diff
@@ -0,0 +1,161 @@
+
+msgid "This section describes important CMake options."
+msgstr "このセクションはCMakeの重要なオプションについて記載しています。"
+
+msgid ""
+"To get more details about installing Groonga from the source by CMake on a "
+"specific environment, find the document for the specific environment from :"
+"doc:`/install`."
+msgstr ""
+":doc:`/install` にある特定環境用のドキュメントに、CMakeでビルドする際のその環"
+"境向けのより詳細な情報があります。特定環境用のドキュメントがある場合は、まず"
+"そちらを参照してください。"
+
+msgid "``-G GENERATOR``"
+msgstr ""
+
+msgid "Specify a generator."
+msgstr "ジェネレータを指定します。"
+
+msgid "This is a mandatory option."
+msgstr "必須オプションです。"
+
+msgid "You can check available generators by ``cmake --help``."
+msgstr "``cmake --help`` で利用可能なジェネレータを確認することができます。"
+
+msgid ""
+"Here is an example of specifying ``Visual Studio 12 2013 Win64`` as a "
+"generator::"
+msgstr ""
+"以下はジェネレータに ``Visual Studio 12 2013 Win64`` を指定する例です。"
+
+msgid "``-DCMAKE_INSTALL_PREFIX``"
+msgstr ""
+
+msgid "Specify a directory/folder to install Groonga."
+msgstr "Groongaのインストール先のディレクトリ/フォルダを指定します。"
+
+msgid ""
+"The default is depending on the system, e.g. `/usr/local` or `C:/Program "
+"Files (x86)/groonga`."
+msgstr ""
+"デフォルトはシステムにより異なります。例えば、 `/usr/local` や `C:/Program "
+"Files (x86)/groonga` です。"
+
+msgid "Here is an example of specifying ``C:\\Groonga`` as an install folder::"
+msgstr "以下は ``C:\\Groonga`` をインストール先に指定する例です。::"
+
+msgid "``-DGRN_WITH_MRUBY``"
+msgstr ""
+
+msgid "Enables mruby support."
+msgstr "mrubyサポートを有効化します。"
+
+msgid ""
+"You can use the :doc:`/reference/sharding` plugin and :doc:`/reference/"
+"commands/ruby_eval` with the mruby support."
+msgstr ""
+"mrubyサポートを有効にすることで :doc:`/reference/sharding` プラグインと :doc:"
+"`/reference/commands/ruby_eval`が使用可能になります。"
+
+msgid "The default is ``off``."
+msgstr "デフォルトは ``off`` です。"
+
+msgid ""
+"Groonga builds bundled mruby if the mruby support is enabled, so you must "
+"install some required libraries to build mruby. See the `mruby compile guide "
+"<https://github.com/mruby/mruby/blob/master/doc/guides/compile.md>`_ for "
+"more details."
+msgstr ""
+"mrubyサポートが有効な場合、Groongaは同梱しているmrubyをビルドします。そのた"
+"め、mrubyをビルドするのに必要なライブラリをインストールする必要があります。詳"
+"細は`mrubyのコンパイルガイド <https://github.com/mruby/mruby/blob/master/doc/"
+"guides/compile.md>`_ を参照してください。"
+
+msgid "Here is an example of enabling the mruby support::"
+msgstr "以下はmrubyサポートを有効にする例です::"
+
+msgid "``-DGRN_WITH_APACHE_ARROW``"
+msgstr ""
+
+msgid "Enables Apache Arrow support."
+msgstr "Apache Arrowサポートを有効化します。"
+
+msgid ""
+"You can use not only Apache Arrow IPC streaming format output but also "
+"multithreading processing that is used in :ref:`select-n-workers` and :doc:`/"
+"reference/functions/query_parallel_or` with the Apache Arrow support."
+msgstr ""
+"Apache Arrowサポートを有効化すると、Apache ArrowのIPC streaming formatが使え"
+"るだけでなく、 :ref:`select-n-workers` や :doc:`/reference/functions/"
+"query_parallel_or` でマルチスレッドで処理できるようになります。"
+
+msgid ""
+"You can install Apache Arrow according to `the official installation "
```

Comment:
```suggestion
"You can install Apache Arrow following to `the official installation "
```
You are going to follow the steps in the documents.

”according to" is more like the document explains how to install. 
You could also say "accordingly".